modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNode.tsx

```diff
@@ -263,11 +263,11 @@ export function ObjectDefinitionNode({
 				/>
 
 				<>
-					{Object.keys(nodeHandleRefs).map((position, index) => (
+					{Object.keys(nodeHandleRefs).map((position) => (
 						<Handle
 							className="lfr-objects__model-builder-node-handle"
-							id={id.toString()}
-							key={index}
+							id={`${id}_${position}`}
+							key={`${id}_${position}`}
```

Comment:
Reason for this change: [see](https://blog.anja-stricker.de/why-using-index-as-key-in-reactjs-map-function-can-cause-problems-and-how-to-fix-them)

Reply:
Hey, @rebsilva! This `key` is only related to the map function. To help avoid unnecessary rendering.

Reply:
okay great! i will approve the PR :blush: good job on searching this and improving the code! :cherry_blossom: